Xwt.XamMac/Xwt.Mac/PopoverBackend.cs

```diff
@@ -279,14 +280,13 @@ void DestroyPopover ()
 		{
 			if (popover != null) {
 				popover.Close ();
-				var controller = popover.Delegate as FactoryViewController;
```

Comment:
Simpler solution would be to dispose of the delegate before closing?

Reply:
It would be, but see https://github.com/mono/xwt/commit/8904459360f4c359f0304c7da77e1c2de9c441bd, we had issues with that not always being called for some reason.